src/arc.ts

```diff
@@ -586,6 +599,90 @@ export function pollForAccountChanges(currentAccountAddress: Address | null, int
   });
 }
 
+async function monitorGraphNodeSubgraph() {
+  if (targetedNetwork() === "ganache") {
+    const query = `{
+      subgraphs {
+        name
+        currentVersion {
+          deployment {
+            id
+            latestEthereumBlockNumber
+            totalEthereumBlocksCount
+            synced
+            failed
+          }
+        }
+      }
+    }`;
+
+    try {
+      const subgraphURL = settings.ganache.graphqlHttpProvider.split("/name/daostack")[0];
```

Comment:
ganache ?

Reply:
that is the name of the key for ganache settings yes

Reply:
why it is needed here? in the production code ?

Reply:
im not sure what you mean. this code checks if the ganache subgraph is running or not. that seems useful for devs to know